### ISSUE TITLE
Assignment 3 Feedback

### ISSUE THREAD

**Issue by egeldenhuys** [egeldenhuys/black-fitch#37]
If you have any feedback regarding the fitchfork results or Black Fitch test cases, leave a comment on this thread.

- Received full marks from fitchfork after passing all Black Fitch test cases for task 1

The `SCENARIO` format is helpful from the user perspective.
Thank you @Quantum-Sicarius and @devosray

**Comment by devosray** [comment]
I got full marks as well for the Assignments thanks to the unit tests. This is really helpful, it really helps. 

I know these tests are adequate but maybe in the future we can also implement integration tests? For Task 2 I tried to do something funky (basically tried to convert die DLL to a SLL in every function call and calling SLL's code) and all the unit tests passed as well as my own unit tests but it failed at Fitchfork due to some integration errors where deleting an element would mess up the array for future calls, but we only tested units on their own. This is my own stupidity I know, but I thought I would post it anyways!

Thanks again, I'll try to write some tests for Prac 8!
